packages/platforms/electron-player/src/renderer/src/classes/ElectronMachine.ts

```diff
@@ -0,0 +1,112 @@
+import { Machine, DeviceInfo } from '@castmill/device';
```

Comment:
The naming of this file in CamelCase does not follow our naming implicit convention for class names, not sure if this is a thing in ElectronJS where camel case may be the standard.

Reply:
changed to kebab case